src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -22,7 +22,8 @@
     },
     "require-dev": {
         "symfony/deprecation-contracts": "^2.5|^3.0",
-        "symfony/error-handler": "^5.4|^6.0"
+        "symfony/error-handler": "^5.4|^6.0",
+        "symfony/polyfill-php81": "^1.27"
```

Comment:
Why do we need this dependency?

Reply:
Because of this: https://github.com/symfony/symfony/pull/48516#discussion_r1049744618 😄

Reply:
Well, okay, it's just a dev dependency, so I guess it's fine.